docs/platforms/javascript/common/best-practices/shared-environments.mdx

```diff
@@ -39,16 +39,18 @@ These best practices are relevant for you if you set up Sentry in one of the fol
 - Libraries
 - Any other scenario where you might have multiple Sentry instances running in the same environment
 
-<Note>
+<Alert level="danger">
 
 When setting up Sentry in a shared environment where multiple Sentry instances may run, you should **not use `Sentry.init()`**, as this will pollute the global state. If your browser extension uses `Sentry.init()`, and a website also uses Sentry, the extension may send events to the website's Sentry project, or vice versa.
 
-</ Note>
+</ Alert>
 
-For such scenarios, you have to set up a client manually as seen in the example below.
-In addition, you should also avoid adding any integrations that use global state, like `Breadcrumbs` or `GlobalHandlers`. Furthermore, some default integrations that use the global state have to be filtered as in the example below.
-As a rule of thumb, it's best to avoid using any integrations and to rely on manual capture of errors only in such scenarios.
+## Shared Environment Setup
 
+For the use cases listed above, set up a client manually as seen in the example below.
+In addition, avoid adding any integrations that use global state, like `Breadcrumbs` or `GlobalHandlers`.
+Furthermore, some default integrations that use the global state have to be filtered as in the example below.
+As a rule of thumb, it's best to avoid using any integrations and to rely on manually capturing errors only in such scenarios.
```

Comment:
```suggestion
In these scenarios, it's a best practice to avoid using any integrations and to rely on manually capturing errors.
```
This is a very opinionated scenario and feel free to reject it, but I got a little lost trying to parse this sentence.